app/serializers/v2/moves_serializer.rb

```diff
@@ -23,14 +24,9 @@ class MovesSerializer
     set_type :moves
 
     INCLUDED_FIELDS = {
-      moves: attributes_to_serialize.keys +
-        %i[profile from_location to_location prison_transfer_reason supplier],
-      profiles: ::V2::ProfileSerializer.attributes_to_serialize.keys + %i[person person_escort_record],
+      # TODO: remove these and replace with conditional relationships in relevant serializer
```

Comment:
I didn't want to increase the scope of this PR further and make potentially breaking changes to person and location serializers as those are used in several other API endpoints which would mean lots of additional testing. Left as a future TODO.